data/human/events.txt

```diff
@@ -1167,12 +1167,95 @@ event "navy occupying the south"
 event "catalytic ramscoop available"
 	outfitter "Deep Sky Basics"
 		"Catalytic Ramscoop"
-
-
-
-event "fw gets catalytic ramscoop"
```

Comment:
Why is this deleted?

Reply:
Dangit, i accidentally deleted it when i pasted over the changes. Thanks for pointing it out, let me add it back in.